src/main/java/de/tum/in/www1/artemis/service/connectors/TextAssessmentConflictService.java

```diff
@@ -0,0 +1,67 @@
+package de.tum.in.www1.artemis.service.connectors;
+
+import static de.tum.in.www1.artemis.service.connectors.RemoteArtemisServiceConnector.authenticationHeaderForSecret;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.exception.NetworkingError;
+import de.tum.in.www1.artemis.service.dto.TextAssessmentConflictRequestDTO;
+import de.tum.in.www1.artemis.service.dto.TextAssessmentConflictResponseDTO;
+
+@Service
+@Profile("automaticText")
+public class TextAssessmentConflictService {
+
+    private final Logger log = LoggerFactory.getLogger(TextAssessmentConflictService.class);
+
+    // region Request/Response DTOs
+    private static class Request {
+
+        public List<TextAssessmentConflictRequestDTO> feedbackWithTextBlock;
+
+        public Long exerciseId;
+
+        Request(List<TextAssessmentConflictRequestDTO> textAssessmentConflictRequestDTOS, long exerciseId) {
+            this.feedbackWithTextBlock = textAssessmentConflictRequestDTOS;
+            this.exerciseId = exerciseId;
+        }
+    }
+
+    private static class Response {
+
+        public List<TextAssessmentConflictResponseDTO> feedbackInconsistencies;
+    }
+    // endregion
+
+    @Value("${artemis.automatic-text.feedback-consistency-url}")
+    private String API_ENDPOINT;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968856)